docs/docs/commands/clients.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 1
```

Comment:
All the commands have `sidebar_position: 1`, please order them in the order we show them when you run `sedge --help`

Reply:
done